.github/workflows/beam_PostCommit_Java_Examples_Dataflow_V2_Java.yml

```diff
@@ -77,7 +77,9 @@ jobs:
         uses: actions/setup-java@v3.8.0
         with:
           distribution: 'temurin'
-          java-version: ${{ matrix.java_version }}
+          java-version: |
```

Comment:
previous setting was not quite what we want. We want beam (java core, runners) compile with Java8 and test compile and run with Java11/17/21.

Reply:
However, we do want the example jar to be compiled on Java11/17/21, see #28120 as followups